.github/workflows/integration_test.yml

```diff
@@ -2,7 +2,7 @@ name: Integration test
 
 env:
   POETRY_VERSION: 1.8.3
-  RIPPLED_DOCKER_IMAGE: rippleci/rippled:2.3.0
+  RIPPLED_DOCKER_IMAGE: rippleci/rippled:develop
```

Comment:
We should point to a specific version otherwise when `develop` gets updated, the integ tests will fail.

Reply:
Wouldn't it be better to have advanced notice that a new rippled version will have issues?

Reply:
I agree with @mvadari. We need to know when the client-libraries are not compatible with rippled's `develop` branch.

Reply:
Yes, historically rippled PRs are merged into develop after a thorough review